.github/workflows/f5-cla.yml

```diff
@@ -0,0 +1,52 @@
+name: F5 CLA
+
+on:
+  issue_comment:
+    types:
+      - created
+  pull_request_target:
+    types:
+      - opened
+      - closed
+      - synchronize
+
+concurrency:
+  group: ${{ github.ref_name }}-cla
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  f5-cla:
+    name: F5 CLA
+    runs-on: ubuntu-22.04
+    permissions:
+      actions: write
+      contents: read
+      pull-requests: write
+      statuses: write
+    steps:
+      - name: Run F5 Contributor License Agreement (CLA) assistant
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have hereby read the F5 CLA and agree to its terms') || github.event_name == 'pull_request_target'
+        uses: contributor-assistant/github-action@f41946747f85d28e9a738f4f38dbcc74b69c7e0e # v2.5.1
+        with:
+          # Any pull request targeting the following branch will trigger a CLA check.
+          branch: "main"
+          # Path to the CLA document.
+          path-to-document: "https://github.com/f5/.github/blob/main/CLA/cla-markdown.md"
+          # Custom CLA messages.
+          custom-notsigned-prcomment: "🎉 Thank you for your contribution! It appears you have not yet signed the F5 Contributor License Agreement (CLA), which is required for your changes to be incorporated into an F5 Open Source Software (OSS) project. Please kindly read the [F5 CLA](https://github.com/f5/.github/blob/main/CLA/cla-markdown.md) and reply on a new comment with the following text to agree:"
+          custom-pr-sign-comment: "I have hereby read the F5 CLA and agree to its terms"
+          custom-allsigned-prcomment: "✅ All required contributors have signed the F5 CLA for this PR. Thank you!"
+          # Remote repository storing CLA signatures.
+          remote-organization-name: "f5"
+          remote-repository-name: "f5-cla-data"
+          path-to-signatures: "signatures/beta/signatures.json"
+          # Comma separated list of usernames for maintainers or any other individuals who should not be prompted for a CLA.
+          allowlist: bjee19, kate-osborn, lucacome, salonichf5, sjberman, bot*
```

Comment:
Any way this could be set to our org groups so we don't have to update this ever again?

Reply:
Unfortunately it's not possible. I think @TasoOneAsia was working on something like a centralized list, but I'm not sure if the action would accept it anyway.

Reply:
That's too bad. This just means that anyone else in the nginx org is gonna have to sign a CLA to commit to an nginx repo, because I don't see us adding everyone to this list...

Reply:
Or maybe the list would be just of people that already "signed" the CLA because the work at F5 and they wouldn't need to be in the allowlist now that I think about it.

Reply:
@sjberman I had originally wanted to investigate making a pull-request [upstream] (https://github.com/contributor-assistant/github-action) and adding a field for `org-allowlist` but haven't had a chance to dig into their source yet.

Reply:
They have an issue for team/orgs https://github.com/contributor-assistant/github-action/issues/100

Wouldn't adding people to the signatures JSON, like they signed the CLA, have the same effect as adding them to the allow list?

Reply:
That should work and I did something similar [migrating](https://github.com/f5/f5-cla-data/commit/2d3cb37fa24bbfff0c3b9435d8f8cd440b391353) `f5/unovis` paper CLAs into the automated tooling. However, I think the comment ID and PR ID have to actually be valid otherwise something goes awry inside the action handling.

Reply:
I guess an option would be to actually just sign the CLA the next time we open a PR instead of having to maintain the allowlist here.

Reply:
That's probably the best approach if core maintainers are pretty variable, at least until this can be solved at the org-wide level.

Reply:
Can we sign for our bots?

Reply:
@sjberman The allowlist has a glob for `bot*` which matches against the GitHub username (i.e dependabot[bot], greenkeeper[bot]). 

If that glob isn't sufficient, bot usernames can be whitelisted individually by adding them to the `allowlist` field.